tools/oesgx/CMakeLists.txt

```diff
@@ -3,11 +3,10 @@
 
 add_executable(oesgx oesgx.c)
 
-target_link_libraries(oesgx oehost)
+target_include_directories(oesgx PRIVATE ${CMAKE_SOURCE_DIR}/include)
```

Comment:
I this this PR could use some comments as to why this change is important for the portability of oesgx. Dmitry and I talked offline about this. I'll write a comment in the conversation describing the problem this PR solves :)